src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -107,7 +107,6 @@
   font-size: $font-size-l;
   line-height: 1.188;
   margin: 14px 0 0 0;
-  overflow-wrap: anywhere;
```

Comment:
In my manual test, I can see that this is no longer wrapping the way we need it to. We probably need to put back any values that had `overflow-wrap: anywhere;`.

Reply:
It's certainly not as before but does that cause problems?

Reply:
It does if you have a long unbroken string.